tests/agent/actions/test_run_metrics_list_action.py

```diff
@@ -94,7 +94,7 @@ def test_run_metrics_list_computes_metric_run_missing_batch_inspector(
     )
 
     # Asserting against mocked class proved challenging
-    action._batch_inspector.compute_metric_list_run.assert_called_once()
+    action._batch_inspector.compute_metric_list_run.assert_called_once()  # type: ignore[attr-defined]  # this is not a mock
```

Comment:
I considered removing this line, but that would mean removing the entire test since this is the only assertion. It seems to work but the type checker complains that there is no `assert_called_once()` 🤔

Reply:
If it's not a mock, do we define this method somewhere?

Reply:
No, that's what the type error is about. I could try to figure out how to mock it, but I'm leaning towards removing this test since I don't understand what it's testing.